charts/islandis/values.dev.yaml

```diff
@@ -345,6 +346,9 @@ api:
     SYSLUMENN_HOST: '/k8s/api/SYSLUMENN_HOST'
     SYSLUMENN_PASSWORD: '/k8s/api/SYSLUMENN_PASSWORD'
     SYSLUMENN_USERNAME: '/k8s/api/SYSLUMENN_USERNAME'
+    DRIVING_LICNESE_BOOK_XROAD_PATH: '/k8s/api/DRIVING_LICNESE_BOOK_XROAD_PATH',
+    DRIVING_LICNESE_BOOK_USERNAME: '/k8s/api/DRIVING_LICNESE_BOOK_USERNAME',
+    DRIVING_LICNESE_BOOK_PASSWORD: '/k8s/api/DRIVING_LICNESE_BOOK_PASSWORD',
```

Comment:
there's a typo in the name btw. :) ... licnese vs. license

Reply:
Fixed.